README.md

```diff
@@ -32,14 +32,14 @@ jobs:
   auto-label:
     runs-on: ubuntu-latest
     steps:
-      - uses: prince-chrismc/label-merge-conflicts-action@v1
+      - uses: prince-chrismc/label-merge-conflicts-action@v1 # Replace this with the latest version
```

Comment:
This is actually the recommended practice for actions

The easiest for consumer is to only follow major, and actions are expected to update that tag to the latest release

```suggestion
      - uses: prince-chrismc/label-merge-conflicts-action@v1
```